clang/test/CXX/drs/cwg27xx.cpp

```diff
@@ -101,6 +101,26 @@ static_assert(!__is_layout_compatible(StructWithAnonUnion, StructWithAnonUnion3)
 #endif
 } // namespace cwg2759
 
+#if __cplusplus >= 202002L
+namespace cwg2770 { // cwg2770: 20
+template<typename T>
+struct B {
+  static_assert(sizeof(T) == 1);
+  using type = int;
+};
+
+template<typename T>
+int f(T t, typename B<T>::type u) requires (sizeof t == 1);
```

Comment:
Using a `sizeof expression` form (without parentheses) seems rather peculiar to me. Is there a reason for this?

Reply:
No, @cor3ntin provides me with it in the above comment, and I just copy-pasted it.